### ISSUE TITLE
Transparant facades showing conduits

### ISSUE THREAD

**Issue by pnieuwkamp** [SleepyTrousers/EnderIO#3332]
#### Issue Description:
There are two conflicting thoughts about transparant facades. There are those who want to view conduits inside a transparant facade (#2737), and there are those who don't (#3226). As those issues describe the matter quite clearly I'm not going to repeat it here :) 

Closing the latter as invalid is, in my point of view, closing your eyes to what a part of your user base wants. For two years (or however long conduit facades have been part of EnderIO) they behaved in one way (consistent with their description: they hide the conduit. There was no need to open a ticket as it did not need fixing: It was working as expected), and the you change them and mark issues pleading to change it back as invalid?

To quote HenryLoenwind from #3226: "great now we get reports telling us that we fixed a bug people liked...", yeah, that's what you get when you "fix" something that wasn't broken in the eyes of (at least some of) your users.

To quote MindForger on that same issue: "i prefere to see the cable through the glass... either way it would be unrealistic!", I prefer NOT to see the cable, I want that block to be glass, not conduit. If I wanted to see the conduit (or didn't care to see it) I wouldn't put a facade on it...

And while there may be ways to build around that in some situations (like, go 10 blocks left, go up, come back 10 block, to go around a window), this is not always possible (and a bit expensive, though that's the least of my worries).

As for realism, most wires and pipes in real life are not 1 by 1 by 1 meter (and if they are, their looks don't matter all that much), so there are loads more options to hide them properly. 

If you want to see a conduit, don't put a facade on it... Conduit facades are used specifically (and pretty much ONLY, as water, lava or mobs are already blocked by the conduit itself) to HIDE conduits, and now they're not hidden, so from my point of view the current situation is most definitely a bug.

**Comment by MatthiasMann** [comment]
It is called a design decision - and I like transparent facades.

**Comment by CrazyPants** [comment]
As you pointed out, either way some people will be unhappy.
As of 1.9 things like torches and anvils can be used to paint facades which makes hiding the conduits silly

**Comment by bookerthegeek** [comment]
Why not have both? Why do we have to have one or the other?

**Comment by pnieuwkamp** [comment]
@CrazyPants : I haven't played 1.9 yet so I'm not quite sure I follow.

There is no longer a need for a painting machine as its function has been superseded by torches and anvils? As much as I try to see the logic behind that, it doesn't tell me what that has to do with how a facade behaves?

On the other hand, if you mean you can paint a facade to look like a torch or an anvil, well, torches are special anyway so I'm not sire what my opinion is, but an anvil is opaque so you couldn't see the conduit anyway (outside of its hitbox or whatever it's called).

@MatthiasMann : I like transparant facades too, but now they're only partially transparant :) I am curious why you would want to facade a conduit if not to hide it though? Is it for consistency when you have a line of conduits going perpendicular through a window?

@bookerthegeek : Though more work for the devs, it would address the issue brilliantly. For instance the current recipe for a facade but with an ender pearl in the center spot to have the conduit take a short hop through "somewhere else" so you don't see it in that block?

**Comment by Mementh** [comment]
to me its HOW i want it to look.

I WANT to hide the cables.. I don't WANT to HAVE to use a wireless light.. since there is LITERLLAY no way to not charge EVERYTHING and only power the lights. 

so please put it back.. or make wireless ligths need to LINK to the power source and not just charge everything.

I literally had a issue where I had my based wired up with wireless lights and somehow was charging EVERYTHING on the server.. 

seriously.. someone 5k blocks away was getting there jetpack charged by me.. 

and i had gone 60k away and was being recharged by a bug.. 

Since then i don't WANT wireless lights.. 

so again its the asthetics.. 


and if its as" 

>  mindforger commented on Mar 15

> sorry to say that, but not beeing able to hide cable properly and using glass to make them invisible is very cheap thinking


if its "CHEAP THINKING" then wireless would not be a issue at all..

maybe have it both ways.. the painter has a option to hide or not.. this way we can have both options and both of us be happy?

**Comment by tterrag1098** [comment]
We understand what you want - but as you said yourself there are just as many (if not more) people who like it the way it is now. And logically - it makes more sense. Why would glass make conduits invisible?

As for the reason WHY you would use a transparent facade, perhaps on a floor, to make it smooth? Or just for aesthetics to give a "see-through" view into your wiring? There are tons of reasons. In the end, if you don't like it, use an opaque block as your facade, or use wireless lights. If there is a bug with those then report/confirm it, it is not relevant here.

In the end it is just a design decision that we have made, and we don't plan to undo it.

**Comment by pnieuwkamp** [comment]
No, it doesn't make sense. Why would a block designed to hide something not hide something?

Where did I say something about lights? And an opaque block in the middle of a glass windows isn't very aesthetic...

So, basically, you've taken away functionality that has been part of the mod since the beginning, what was working as designed, which bothered nobody for three years, and you don't give an alternative, making the description of the block inaccurate (to the least) in the process. From your very own wiki: "The conduit facade allows you to hide conduits behind a block". Well, they don't... And yes, of course you can update the wiki, but you are just fundamentally changing what the block does. It's like you change the SAG mill to smelt ore to ingots instead of dusts and call it a day...

**Comment by AnrDaemon** [comment]
"Cover" not necessarily means "hide".

**Comment by pnieuwkamp** [comment]
Indeed it does not (though that is by far the most common use), so it's a good thing we're discussing something designed to hide instead of to cover :P

**Comment by AnrDaemon** [comment]
Since topic is closed already, I see no point in dragging this… discussion.
The only possible followup is topic lock, and I really don't want to get that kind of attention.

**Comment by tterrag1098** [comment]
>Why would a block designed to hide something not hide something?

Because the block is transparent? Because you painted it as such?

**Comment by tterrag1098** [comment]
I'm done discussing this. The decision is final, the behavior will not be reversed. All devs are in agreement. You are welcome to suggest alternatives in a separate issue, but "transparent facades show conduits" is *not* a bug.
